package.json

```
@@ -38,8 +39,7 @@
     "canary:bumpVersion": "yarn lerna version `yarn --silent canary:version` --exact --no-push --yes",
     "canary:publish": "yarn lerna publish from-package --dist-tag canary --yes --no-verify-access",
     "changelog": "lerna-changelog",
-    "postinstall": "run-p postinstall:**",
-    "postinstall:main": "yarn lock:update && yarn build:packages",
+    "postinstall": "run-p postinstall:\"**\"",
```

are you sure about this?

i think this should be
```suggestion
    "postinstall": "run-p postinstall:**",
    "postinstall:main": "yarn build:packages",
```

---

I'm still investigating this. It seems the `postinstall` hooks aren't even executed

---

> It seems the postinstall hooks aren't even executed

They are, but only when the dependency tree changes.

> postinstall is called after a package's dependency tree changes are written to the disk -- e.g. after a dependency or transitive dependency is added, removed, or changed
https://yarnpkg.com/advanced/lifecycle-scripts